### PR TITLE
update C# example

### PR DESCRIPTION
**Pull request by JssDWt**
Needs the fee changes still

### CODE REVIEW

File: src/guide/receive_onchain.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ if swapInfo, err := sdkServices.ReceiveOnchain(); err != nil {
 ```cs
 try 
 {
-    var swapInfo = sdk.ReceiveOnchain();
+    var swapInfo = sdk.ReceiveOnchain(new ReceiveOnchainRequest(null));
```
Comment (roeierez) [inline src/guide/receive_onchain.md:105]: Do we have to pass null here? We use default value in the udl.

**Review by roeierez — CHANGES_REQUESTED** [review]
Looks good!  Dropped a small comment regarding default values.

File: src/guide/receive_onchain.md (hunk 1 above)
Comment (JssDWt) [reply src/guide/receive_onchain.md:105]: Yes, we do:
Scherm­afbeelding 2023-09-03 om 20 45 47

Comment (roeierez) [reply src/guide/receive_onchain.md:105]: It is in this PR that is going to be merged with the new docs:
https://github.com/breez/breez-sdk/pull/416/files#diff-5f8d23b285fe7996da0509e9fbc2496bfc9d3ea4040a4702cbe85cfc38be09eaR314

Comment (JssDWt) [reply src/guide/receive_onchain.md:105]: Ah, great

Comment (JssDWt) [reply src/guide/receive_onchain.md]: Done

**Comment by JssDWt** [comment]
Points to ok300's PR now.

**Pull request merged**